### ISSUE TITLE
invalid sha1 on maven central for play-json-naming_2.11-1.0.0.pom

### ISSUE THREAD

**Issue by edvakf** [tototoshi/play-json-naming#2]
Can you check your release?

```
...
[info] Resolving com.github.tototoshi#play-json-naming_2.11;1.0.0 ...
[warn] problem while downloading module descriptor: https://repo1.maven.org/maven2/com/github/tototoshi/play-json-naming_2.11/1.0.0/play-json-naming_2.11-1.0.0.pom: invalid sha1: expected=1600123149a624d2d324a28b12a4446ca142f2a1 computed=a5d156e3c8f9b1c7484c60fed82cdeec5057
f68a (536ms)
[warn]  module not found: com.github.tototoshi#play-json-naming_2.11;1.0.0
...
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: com.github.tototoshi#play-json-naming_2.11;1.0.0: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]
[warn]  Note: Unresolved dependencies path:
...
[error] (root/*:update) sbt.ResolveException: unresolved dependency: com.github.tototoshi#play-json-naming_2.11;1.0.0: not found
[error] Total time: 7 s, completed Oct 4, 2015 7:57:40 AM
```

**Comment by tototoshi** [comment]
Published 1.0.0 once again. 

https://repo1.maven.org/maven2/com/github/tototoshi/play-json-naming_2.11/1.0.0/

**Comment by edvakf** [comment]
Worked. Thank you.
